### PR TITLE
Docking Layout

### PR DESCRIPTION
**Pull request by indigo-san**
## Description
This PR introduces a docking layout.
Additionally, the previous page navigation has been removed, and the pages that existed before are now displayed as independent dialogs.

## Breaking changes
- ToolTabExtension

### CODE REVIEW

**Comment by yuto-trd** [comment]
**TODO**
- [ ] ~~Floating tool window~~

Do it in another PR.

File: src/Beutl/Pages/ExtensionsPage.axaml.cs [hunk 1]
```diff
@@ -63,10 +63,11 @@ private void Search_Click(object? sender, RoutedEventArgs e)
 
     private void OpenSettings_Click(object? sender, RoutedEventArgs e)
     {
-        if (this.FindLogicalAncestorOfType<MainView>() is { DataContext: MainViewModel viewModel })
-        {
-            viewModel.SelectedPage.Value = viewModel.SettingsPage;
-        }
+        // TODO: 設定ウィンドウを開く
```
Comment (indigo-san) [inline src/Beutl/Pages/ExtensionsPage.axaml.cs:66]: TODO

File: src/Beutl/ViewModels/EditViewModel.cs
Comment (indigo-san) [inline src/Beutl/ViewModels/EditViewModel.cs:1]: Split into multiple files

File: src/Beutl/ViewModels/SettingsPageViewModel.cs
Comment (indigo-san) [inline src/Beutl/ViewModels/SettingsPageViewModel.cs:1]: Rename

File: src/Beutl/Views/EditorHostFallback.axaml.cs [hunk 2]
```diff
@@ -218,7 +212,7 @@ private void InitRecentItems()
     {
         ViewConfig viewConfig = GlobalConfiguration.Instance.ViewConfig;
 
-        IObservable<int> filter = FilterComboBox.GetObservable(SelectingItemsControl.SelectedIndexProperty);
+        IObservable<int> filter = AvaloniaObjectExtensions.GetObservable(FilterComboBox, SelectingItemsControl.SelectedIndexProperty);
```
Comment (indigo-san) [inline src/Beutl/Views/EditorHostFallback.axaml.cs:215]: Use extension method

File: src/Beutl/Views/MainView.axaml [hunk 3]
```diff
@@ -1,19 +1,14 @@
...
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:icons="using:FluentIcons.FluentAvalonia"
-             xmlns:interactivity="using:Avalonia.Xaml.Interactivity"
              xmlns:lang="using:Beutl.Language"
-             xmlns:local="using:Beutl.Views"
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-             xmlns:pages="using:Beutl.Pages"
              xmlns:ui="using:FluentAvalonia.UI.Controls"
-             xmlns:uip="using:FluentAvalonia.UI.Controls.Primitives"
+             xmlns:views="clr-namespace:Beutl.Views"
```
Comment (indigo-san) [inline src/Beutl/Views/MainView.axaml:9]: Use `using:Beutl.Views`

File: src/Beutl/Views/Tools/SourceOperatorsTab.axaml.cs [hunk 4]
```diff
@@ -69,13 +69,13 @@ private void Drop(object? sender, DragEventArgs e)
...
-        }
+        // if (e.Data.Contains(KnownLibraryItemFormats.SourceOperator))
+        // {
+        //     e.DragEffects = DragDropEffects.Copy | DragDropEffects.Link;
+        // }
+        // else
+        // {
+        //     e.DragEffects = DragDropEffects.None;
+        // }
     }
```
Comment (indigo-san) [inline src/Beutl/Views/Tools/SourceOperatorsTab.axaml.cs:80]: Delete this method

File: src/Beutl/Views/EditView.axaml [hunk 5]
```diff
@@ -1,116 +1,161 @@
...
+            <dock:DockArea LocalizedName="BottomLeft"
+                           Location="LeftLowerBottom"
+                           SideBar="LeftSideBar"
+                           Target="LeftContent"
+                           View="BottomLeftRightArea" />
+            <dock:DockArea LocalizedName="BottomRight"
+                           Location="RightLowerBottom"
+                           SideBar="RightSideBar"
+                           Target="RightContent"
+                           View="BottomLeftRightArea" />
```
Comment (indigo-san) [inline src/Beutl/Views/EditView.axaml:89]: Localize

**Pull request merged**